### ISSUE TITLE
Peer feedback from Team SSTL

### ISSUE THREAD

**Issue by hmk792** [NumEconCopenhagen/projects-2021-hold-j1#1]
Dear Team-J1

We have been reviewing your data project and tried to provide you with some constructive comments for you to work with. Not all comments are positive, but we just want you to have the most constructive feedback :-) . We have answered the issues in the template down below and overall we think you have made a good data project, but the structure and setup of the project needs a bit of work. 

Before we begin we need to point out to you that it is not possible to just clone your repository and then run the code. This is due to the fact that your data set is called 'data3.xlsx' in the code, but the file in your repository is called 'data3 - assignment 2.xlsx'. We just renamed the file and then ran the code, but you should be attentive to this! I would also suggest you to clean up your repository and delete files you don't use in the data project, and remember to fill out the README.md file. 

**1. The best part of the project was:**

The idea to investigate differences in income levels across municipalities and across gender. I think it is good project and you have good data for the analysis. The graphs in the analysis are also good.  

**2. The hardest part of the project to understand was:** 

Your description of the data in the last part of section 3.1 was very hard to understand. You write "If we consider the mean for the average income for all the municipalities the past 10 years, wee see that the mean for the average income has increased with 47.832.000 dkk. If we also consider the minimum and maximum it is clear from the table that the municipalities who earned the most has increased more than the bottom municipalities, as wee see and increase for the maximum at 146.803.000, whereas the minimum only increased with 35.164.000. The maximum municipalities avrage income is increased more than twice compaaring to the minimum."

All of these numbers must be incorrect, and we also don't find them anywhere? The average income level are in  thousands and you are writing about increases in millions (Also be attentive to the meaning of , and .). It is also not possible to see that the municipalities who earned the most has increased more than the bottom municipalities, because there are no indicators of placement of the different municipalities in this table. 

**3. This part of the projet could be better documented:** 

Your code. You don't apply the # to comment on the different lines of code (except from when you import data) and that makes it hard to follow and understand your code. It is also a big help to yourself if you write comments in the code explaining what you have done. 

**4. An idea for an improvement/clarification could be:** 

It looks like you tried to give structure to the project by making a section called introduction, but sadly it is unfilled and we first understand what the project is about after you start loading and cleaning data. We would suggest you to spend some more time on writing an introduction and explain the project, so that the reader can follow the project from the beginning. The project also need proof-reading. 

Maybe it would also be an idea to restructure your project, so that you first import all data and clean it and then describe and analyse the data. 

**5. An idea for an extension could be:** 

Since you have downloaded data from Statistics Denmark an obvious extension could be to import the data using an API. You could even check out our project, since we also work with data for average income levels across municipalities imported from Statistics Denmark via API. 

I think you should try and use a widget, e.g. you could make a widget were you could explore the data for every municipality by having a dropdown box, where you can choose a municipality and then see a graph showing average income (total, men, women) in the period. (Again, you can find inspiration from our project here.)

**Comment by bjs201** [comment]
Overall, the assignment works well but it would have been great with an introduction in the readme as well given that this is the pre-defined format in the course. However, that is easy to find in the actual code itself. Here are a few comments on your project.

1. Is it straightforward to run the code?
The overarching comment here is the difficulties replicating the code, thus testing if it actually works. There seems to have been some sort of error in finding the location of the data? Browsing through the code, it seems intuitively structured as the narrative of the assignment moves along. Good job! :D

2. Is it easy to understand what the code is doing?
There is an element of self-criticism in answering this question due to my own limited experience from coding in Python. Obviously, there are several ways to reach your objective and while our code may not be identical, I believe that overall the code is fairly easy to understand - especially in conjunction with the output. 

3. Are the results clearly explained?
This is actually one of the best part of the paper. The graphs are simple and work well, which yields a somewhat self-explanatory result. However, the description towards the ending of the assignment becomes somewhat difficult to understand. I could be me but I am not quite sure what the result is when reading 3.1. 

4. Do you have some ideas for improvements or extensions?
The overarching comment is that there is always room for improvement. Given my limited experience with coding Python, I find it difficult to bring something code-savvy to the table. However, there is room for improvement in the structure. A relatively easy way to optimize your assignment is to find a structure for answering each question that works for you. Then ask yourself, would a noob understand it? Once you have done this, you can apply it to each question in the assignment.

I hope this was somewhat a value add.

Great job :)
